src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx

```diff
@@ -107,10 +111,16 @@ ANR profiles shows the distribution of attributes across all your ANR occurrence
     title="anr profiles"
     alt="screenshot of anr profiles"
     src={mobileAnrProfiles}
+    style="float: left;"
 />
 
 ## Disable ANR monitoring [#disable-anr-monitoring]
 
+<Callout variant="important">
+Server-side configuration always take precedence over local agent configuration.
+For changes to be applied on end-user devices, it may take up to 10 minutes for mobile apps with 100,000 or fewer concurrently active devices. For applications with very large number of concurrently active devices, it may take several hours.
```

Comment:
```suggestion
For mobile apps with <100,000 concurrently active devices, it may take up to 10 minutes to apply changes to end-user devices. For apps with >100,000 active devices, it may take several hours.
```